modules/virt-supported-features-for-virtual-storage-matrix.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc
+
+[id="virt-supported-features-for-virtual-storage-matrix_{context}"]
+= Supported features for {VirtProductName} virtual storage
+
+.{VirtProductName} virtual storage feature support matrix
+[cols="40%,15%,15%,15%,15%"]
+|===
+||Virtual machine live migration  |Host-assisted virtual machine disk cloning |Storage-assisted virtual machine disk cloning |Virtual machine import
+
+|{product-title} container storage: RBD block volumes
+|Supported
+|Supported
+|Supported
+|Supported
```

Comment:
please note that we just figured that V2V doesnt work with RWX storage, so please mark it as a 'no'

Reply:
I am not sure whether I get you comment correctly.
Actually we set accessmode depending on source configuration. You can find the logic [here](https://github.com/kubevirt/vm-import-operator/blob/c128de0cd61d99af2b84596a1c1319fa2a175b20/pkg/providers/ovirt/mapper/mapper.go#L228). The issue you are referring to was with Cinder which do not support `ReadWriteMany`.

Reply:
Yeah, not supporting import to rwx (if the k8s storage supports it) would be a bug needing a code fix.

Reply:
@aglitke Good to know the clear description of the later comment. I prefer we say like "Block mode PVC on a Ceph RBD device" or "Ceph RBD block-mode volumes". Maybe with or without Ceph is not a big deal somewhat, because when people mentioned RBD, it's absolutely the Ceph RBD and couldn't be other RBD, but I figure that it‘s profitable to highlight RedHat product name in any occasion.

Reply:
Changed "Virtual machine import" for "OpenShift Container Platform container storage: Ceph RBD block-mode volumes" to **No**.